jme3-core/src/plugins/java/com/jme3/texture/plugins/TGALoader.java

```diff
@@ -38,6 +38,12 @@
 import com.jme3.texture.Image;
 import com.jme3.texture.Image.Format;
 import com.jme3.util.BufferUtils;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.awt.image.DataBuffer;
+import java.awt.image.DataBufferByte;
+import java.awt.image.Raster;
```

Comment:
`jme3-core` should not reference the `java.awt` package, because it is not available on Android. 
`java.awt` is allowed to be referenced in desktop only projects, like `jme3-desktop` and `jme3-lwjgl`.